tests/TestCase/Controller/Component/AuthenticationComponentTest.php

```diff
@@ -105,7 +102,7 @@ public function testGetAuthenticationService()
     {
         $service = new AuthenticationService();
         $request = $this->request->withAttribute('authentication', $service);
-        $controller = new Controller($request, $this->response);
+        $controller = new Controller($request);
```

Comment:
These are the places where a response was passed to Controller. Not sure why it was done this way since removing it doesn't affect anything.

Reply:
Most likely copy paste damage.